tools/db/move-team/package.yaml

```
@@ -0,0 +1,64 @@
+defaults:
+  local: ../../../package-defaults.yaml
+name: move-team
+version: '1.0.0'
+synopsis: Export a team from one backend, or import it into another
+category: Network
+author: Wire Swiss GmbH
+maintainer: Wire Swiss GmbH <backend@wire.com>
+copyright: (c) 2018 Wire Swiss GmbH
+license: AGPL-3
+ghc-options:
+- -funbox-strict-fields
+- -threaded
+- -with-rtsopts=-N
+- -with-rtsopts=-T
+- -rtsopts
+dependencies:
+- base
+- aeson
+- attoparsec
+- brig-types
+- bytestring
+- bytestring
+- bytestring-conversion
+- cassandra-util
+- conduit
+- containers
+- cql
+- directory
+- extended
+- filepath
+- galley
+- galley-types
+- imports
+- iproute
+- lens
+- megaparsec
+- mtl
+- optparse-applicative
+- process
+- parser-combinators
+- raw-strings-qq
+- stache
+- text
+- time
+- tinylog
+- types-common
+- unliftio
+- uuid
+- vector
+- wire-api
+library:
+  source-dirs: src
+
+executables:
+  move-team:
+    main: Main.hs
+    source-dirs: src
+
+  gen-table-types:
+    main: Run.hs
+    source-dirs: gen-table-types/src
+    dependencies:
+      - move-team
```

I think it's good to depend both executables on the library to save redundant recompilations.  Not sure, though.  Anyway this could be more symmetrical (which is good iff you like symmetry):

```suggestion
  move-team:
    main: MoveTeam.hs
    source-dirs: app
    dependencies:
      - move-team

  gen-table-types:
    main: MoveTeamGenerate.hs
    source-dirs: app
    dependencies:
      - move-team
```

---

Done

---

I meant something different.  Couldn't get it to work, because `main-is` doesn't seem to work particularly well with `hpack`, but I had this: 9d0d42978